README.md

```diff
@@ -94,10 +94,9 @@ issues](./documentation/known_issues.markdown) that you should be aware of.
 
 If want to play with our code, these documents should prove useful:
 
--   [Running Puppet Server from
-    source](./documentation/dev_running_from_source.markdown)
--   [Debugging](./documentation/dev_debugging.markdown)
--   [Puppet Server subcommands](./documentation/subcommands.markdown)
+-   [Running Puppet Server from source](https://puppet.com/docs/puppet/6.19/server/dev_running_from_source.html)
+-   [Debugging](https://puppet.com/docs/puppet/6.19/server/dev_debugging.html)
+-   [Puppet Server subcommands](https://puppet.com/docs/puppet/6.19/server/subcommands.html)
```

Comment:
hmm I was gonna suggest we use `latest` in these urls instead of a hard-coded version, but I guess that should only apply to main. Just thinking that these could get out-of-date quickly.

Reply:
Ugh, I was going to use latest but it updated automatically with a redirect. I'll fix it.